index.js

```diff
@@ -14,6 +14,10 @@ alfy.fetch('https://api.npms.io/v2/search', {
 		.filter(x => x.package.name.length > 1)
 		.map(x => {
 			const pkg = x.package;
+			const date = new Date(pkg.date);
+			const months = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'];
+			const dates = months.concat(['13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31']);
+			const publishDate = `${date.getFullYear()}-${dates[date.getDate() - 1]}-${months[date.getMonth()]}`;
```

Comment:
Shouldn't we rather use something like [date-fns#format](https://date-fns.org/docs/format) instead of rolling our own?